clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/StandardGceAttributeValidator.groovy

```diff
@@ -191,6 +193,15 @@ class StandardGceAttributeValidator {
     return result
   }
 
+  def validateBetweenZeroAndOneDouble(double value, String attribute) {
+    def result = true
+    if (value < 0 || value > 1) {
```

Comment:
You should probably write these as `0d` and `1d` to make sure the comparison isn't coerced into the wrong type.